### PR TITLE
fixes for mongodb 5.0 & 6.0 upgrade scripts

### PR DESCRIPTION
**Pull request by ihaardik**

### CODE REVIEW

File: bin/upgrade/DEV/upgrade.mongo.50.sh [hunk 1]
```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
 
 #check if authentication is required
-isAuth=$(mongo --eval "db.getUsers()" | grep -v "connecting" | grep "auth")
```
Comment (kaanklky) [inline bin/upgrade/DEV/upgrade.mongo.50.sh:4]: set `isAuth` to `0` here and remove `else` part of next `if` block

File: bin/upgrade/DEV/upgrade.mongo.50.sh [hunk 2]
```diff
@@ -45,13 +49,21 @@ if [ -x "$(command -v mongo)" ]; then
...
 
     if [ -f /etc/lsb-release ]; then
+        UBUNTU_YEAR="$(lsb_release -sr | cut -d '.' -f 1)";
+        if [ "$UBUNTU_YEAR" == "22" ]; then
+            sed -i "s/#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf
+        fi
         #uninstall mognodb
-        apt-get remove -y mongodb-org mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools
+        apt-get remove -y mongodb-org mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools python3-apt
+        apt-get install python3-apt -y
```
Comment (kaanklky) [inline bin/upgrade/DEV/upgrade.mongo.50.sh:66]: follow the convention and put `-y` after `install`

File: bin/upgrade/DEV/upgrade.mongo.50.sh (hunk 1 above)
Comment (ihaardik) [reply bin/upgrade/DEV/upgrade.mongo.50.sh:4]: updated the code block 👍

File: bin/upgrade/DEV/upgrade.mongo.60.sh [hunk 3]
```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
 
 #check if authentication is required
-isAuth=$(mongosh --eval "db.getUsers()" | grep -v "connecting" | grep "auth")
```
[the remark this answers is not in the document]
Comment (ihaardik) [reply bin/upgrade/DEV/upgrade.mongo.60.sh:4]: updated the code block 👍

File: bin/upgrade/DEV/upgrade.mongo.50.sh (hunk 2 above)
Comment (ihaardik) [reply bin/upgrade/DEV/upgrade.mongo.50.sh]: made the changes according to convention

**Review by kaanklky — APPROVED** [review]

**Pull request merged**